### PR TITLE
Fixes issue with repleaderboard

### PR DESCRIPTION
**Pull request by Zcamt**
Fixes the small issue of everyone being number 1 on the leaderboard

### CODE REVIEW

**Review by Nuckerr — APPROVED** [review]
who wrote the original code?? (probably me, looks wacky af looks a LOT better)

**Comment by Nuckerr** [comment]
Going to merge this after workflows branch is merged

**Comment by Zcamt** [comment]
> who wrote the original code?? (probably me, looks wacky af looks a LOT better)

Nope was me haha, got to show my inexperience in kotlin lol

**Pull request merged**
